daisy/step_delete_resources.go

```diff
@@ -265,7 +278,22 @@ func (d *DeleteResources) run(ctx context.Context, s *Step) dErr {
 		}(d)
 	}
 
-	// Delete disks after instances.
+	// Delete subnetworks after instances.
```

Comment:
You'll need to have a separate wait after deleting subnetworks, subnetworks can't be deleted until all instances that reference them are deleted (done with the wait above), but then networks can't be deleted until all subnetworks are deleted

Reply:
done. I took this chance to make it an auxiliary function to avoid triplicating the code

Reply:
Thanks, I was going to mention something like that but figured we could clean it up later.